float64 X
float64 Y
float64 Z
float64 OR_X
float64 OR_Y
float64 OR_Z
float64 OR_W